mep/books/admin.py

```diff
@@ -39,7 +42,26 @@ class ItemAdmin(admin.ModelAdmin):
             )
         })
     )
-    readonly_fields = ('mep_id',)
+    readonly_fields = ('mep_id', 'borrow_count')
+
+    def get_queryset(self, request):
+        '''Annotate the queryset with the number of borrows for sorting'''
+        qs = super(ItemAdmin, self).get_queryset(request)
+        qs = qs.annotate(Count('borrow'))
+        return qs
+
+    def borrow_count(self, obj):
+        '''Display the borrow count as a link to view associated borrows'''
+        return format_html(
+            '<a href="{}">{}</a>',
+            # TODO make this more elegant?
+            reverse('admin:accounts_borrow_changelist') + '?item__id__exact=' + str(obj.id),
```

Comment:
You should move the hard-coded querystring up into the format string.  No need to use `+` (which is inefficient for strings),  just pass multiple args.  Looks like you can use something like `{0!s}` for the id instead of calling `str` - https://docs.python.org/3/library/string.html#formatstrings